server/controllers/stock/reports/stock/expiration_report.js

```diff
@@ -46,11 +46,11 @@ async function stockExpirationReport(req, res, next) {
     // get the lots for this depot
     const lots = await stockCore.getLotsDepot(options.depot_uuid, options);
 
-    // get the lots that are "at risk"
-    const risky = lots.filter(lot => (lot.S_RISK < 0 && lot.lifetime > 0));
+    // get the lots that are "at risk" of expiring
+    const risky = lots.filter(lot => (lot.near_expiration && lot.lifetime > 0));
 
     // get expired lots
-    const expired = lots.filter(lot => (lot.S_RISK <= 0 && lot.expiration_date <= today));
+    const expired = lots.filter(lot => (lot.flags.expired && lot.expiration_date <= today));
```

Comment:
You probably don't need the expiration date check - the `expired` flag will already do that for you.

Reply:
Fixed